features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/MessagesPresenter.kt

```diff
@@ -157,9 +155,9 @@ class MessagesPresenter @AssistedInject constructor(
             mutableStateOf(false)
         }
 
-        LaunchedEffect(currentSessionIdHolder.current) {
+        LaunchedEffect(syncUpdateFlow) {
```

Comment:
If you want the check to be done at each sync, you should use `syncUpdateFlow.value` instead

Reply:
Ah yes, good catch. So there is also a pb here https://github.com/element-hq/element-x-android/blob/develop/features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/MessagesPresenter.kt#L168 ?

Reply:
https://github.com/element-hq/element-x-android/pull/2313/commits/05cc526be98f08f4560c6a1795c091996bbcd2e6